docs/content/integrations/airbyte-cloud.mdx

```diff
@@ -0,0 +1,234 @@
+
+## Step 2: Launching Airbyte Cloud syncs using Dagster
+
+In order to create software-defined assets for your Airbyte Cloud connections, you will first need to determine the connection IDs for each of the connections you would like to build assets for. The connection ID can be seen in the URL of the connection page when viewing the Airbyte Cloud UI.
+
+<center>
+  <Image
+    alt="Screenshot of the Airbyte UI in a browser, showing the connection ID in the URL."
+    src="/images/integrations/airbyte/airbyte_connection_ui.png"
+    width={3102}
+    height={1746}
+  />
+</center>
+
+Then, supply the connection ID and the list of tables which the connection creates in the destination to `build_airbyte_assets`:
+
+```python startafter=start_manually_define_airbyte_assets endbefore=end_manually_define_airbyte_assets file=/integrations/airbyte/airbyte.py dedent=4
+from dagster_airbyte import build_airbyte_assets
+
+airbyte_assets = build_airbyte_assets(
+    connection_id="87b7fe85-a22c-420e-8d74-b30e7ede77df",
+    destination_tables=["releases", "tags", "teams"],
+)
+```
+
+#### Adding a resource
+
+Manually built Airbyte assets require an `airbyte_resource`, which defines how to connect and interact with your Airbyte Cloud instance.
+
+We can add the Airbyte resource we configured above to our Airbyte assets by doing the following:
+
+```python startafter=start_airbyte_manual_config endbefore=end_airbyte_manual_config file=/integrations/airbyte/airbyte.py dedent=4
+from dagster_airbyte import build_airbyte_assets
+
+from dagster import with_resources
+
+airbyte_assets = with_resources(
+    build_airbyte_assets(
+        connection_id="87b7fe85-a22c-420e-8d74-b30e7ede77df",
+        destination_tables=["releases", "tags", "teams"],
+    ),
+    # Use the airbyte_instance resource we defined in Step 1
+    {"airbyte": airbyte_instance},
+)
+```
+
+---
+
+## Step 3: Adding downstream assets
+
+<Note>
+  Looking to orchestrate Airbyte with dbt? Check out our{" "}
+  <a href="https://github.com/dagster-io/dagster/tree/master/examples/assets_modern_data_stack">
+    Modern Data Stack example
+  </a>{" "}
+  and our <a href="/integrations/dbt">dbt integration docs</a>.
+</Note>
+
+Once you have loaded your Airbyte Cloud assets into Dagster, you can create assets which depend on them. These can be other assets pulled in from external sources such as [dbt](/integrations/dbt) or assets defined in Python code.
+
+In this case, we have an Airbyte connection which is storing data in the `stargazers` table in our Snowflake warehouse. We specify the output [IO manager](/concepts/io-management/io-managers) to tell downstream assets how to retreive the data.
+
+```python startafter=start_add_downstream_assets endbefore=end_add_downstream_assets file=/integrations/airbyte/airbyte.py dedent=4
+import json
+from dagster import asset, Definitions
+from dagster_airbyte import load_assets_from_airbyte_instance
+
+airbyte_assets = load_assets_from_airbyte_instance(
```

Comment:
should this be the build command for airbyte cloud instead of load?

Reply:
yes, fixed!